src/Auxiliary/Aux_Check_Parameter.cpp

```diff
@@ -1093,8 +1120,47 @@ void Aux_Check_Parameter()
       Aux_Error( ERROR_INFO, "ELBDM_BASE_SPECTRAL must work with SUPPORT_FFTW !!\n" );
 #  endif
 
-#  if ( WAVE_SCHEME == WAVE_FD )
+// check hybrid scheme parameters for errors
+#  if ( ELBDM_SCHEME == ELBDM_HYBRID )
+
+   if ( INIT_SUBSAMPLING_NCELL > 1 )
+      Aux_Error( ERROR_INFO, "ELBDM_HYBRID currently does not support INIT_SUBSAMPLING_NCELL > 1 !!\n" );
```

Comment:
Add the comment `###REVISE: support rescaling phase for subsampling` around line 158 in `ELBDM_Init_ByFunction_AssignData.cpp`.